tensorflow/compiler/xla/service/cpu/simple_orc_jit.cc

```diff
@@ -275,7 +293,11 @@ bool RegisterKnownJITSymbols() {
   REGISTER_LIBM_SYMBOL(scalbln, double (*)(double, long));
   REGISTER_LIBM_SYMBOL(scalbn, double (*)(double, int));
   REGISTER_LIBM_SYMBOL(sin, double (*)(double));
+#if defined(__APPLE__)
```

Comment:
Sorry for not noticing this earlier (and this is a minor point), but we should consistently use either `#ifdef` or `#if defined()`.

Reply:
no worries. just updated.